README.md

```diff
@@ -430,3 +407,7 @@ for, then the README is missing important information that should be added in.
 Please [file an issue](https://github.com/Pomax/react-onclickoutside/issues) with a request for additional documentation,
 describing what you were hoping to find in enough detail that it can be used to
 write up the information you needed.
+
+## In addition
+
+To avoid findDOMNode use "outsideNodeRef" props from a wrapping component and set it to element as ref.
```

Comment:
People's own code doesn't use `findDOMNode`, can you explain why this section would be needed?

Reply:
Developers should set outsideNodeRef themselves to avoid `findDOMNode` issue

Reply:
then let's move this a whole lot up, because that makes this an important note that should be part of the "how to use" instructions, rather than sneaking it in at the very end of the readme.

We should a slighlty longer explanation and then I'd say let's put this directly below the "Ensuring there is a click handler" section, with a code example, and an explanation of why people want to do this, with a directly quoted error so that if folks run into that error, and search the readme, they find the section that explains how to fix it.